medusa/server/api/v2/cache.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
+"""Request handler for shows."""
+from tornado.escape import json_decode
+from .base import BaseRequestHandler
+from .... import db
+
+
+class SceneExceptionHandler(BaseRequestHandler):
+    """Shows request handler."""
+
+    def set_default_headers(self):
+        """Set default CORS headers."""
+        super(SceneExceptionHandler, self).set_default_headers()
+        self.set_header('Access-Control-Allow-Methods', 'GET, OPTIONS, PUT, POST')
+
+    def get(self, *args, **kwargs):
+        """Query scene_exception information.
+
+        :param show_indexer_id: The indexer id. Like 1 for tmdb and 3 for tvmaze.
+        :param show_id:
+        :type show_indexer_id: str
+        :param season:
+        """
+        try:
+            exception_id = int(kwargs.pop('row_id', 'throwme'))
+        except (ValueError, TypeError):
+            return self.api_finish(status=400, error="Require a valid exception row id.")
+
+        indexer = self.get_query_argument('indexer', None)
+        indexer_id = self.get_query_argument('indexer_id', None)
+        season = self.get_query_argument('season', None)
+
+        cache_db_con = db.DBConnection('cache.db')
+        if indexer and indexer_id and season:
+            sql_base = b'SELECT * FROM scene_exceptions WHERE indexer = ? AND indexer_id = ?'
+            params = [indexer, indexer_id]
+            if season:
+                sql_base += b' AND season = ?'
+                params.append(season)
+        else:
+            sql_base = b'SELECT * FROM scene_exceptions'
+            params = []
+            if exception_id:
+                sql_base += ' WHERE exception_id = ?'
+                params.append(exception_id)
+
+        if sql_base:
+            exceptions = cache_db_con.select(sql_base, params)
+        else:
+            self.api_finish(data={},error="Error, need to provide")
```

Comment:
We should be using correct HTTP codes for all errors. 200 is only to be sent if it was actually successful.